libraries/freertos_plus/standard/freertos_plus_tcp/include/FreeRTOS_IP.h

```diff
@@ -114,19 +150,24 @@ typedef enum eNETWORK_EVENTS
 	eNetworkDown	/* The network connection has been lost. */
 } eIPCallbackEvent_t;
 
-typedef enum ePING_REPLY_STATUS
-{
-	eSuccess = 0,		/* A correct reply has been received for an outgoing ping. */
-	eInvalidChecksum,	/* A reply was received for an outgoing ping but the checksum of the reply was incorrect. */
-	eInvalidData		/* A reply was received to an outgoing ping but the payload of the reply was not correct. */
-} ePingReplyStatus_t;
+#if ( ipconfigSUPPORT_OUTGOING_PINGS == 1 )
+	typedef enum ePING_REPLY_STATUS
+	{
+		eSuccess = 0,		/* A correct reply has been received for an outgoing ping. */
+		eInvalidChecksum,	/* A reply was received for an outgoing ping but the checksum of the reply was incorrect. */
+		eInvalidData		/* A reply was received to an outgoing ping but the payload of the reply was not correct. */
+	} ePingReplyStatus_t;
```

Comment:
Actually, those modules should not refer to `ePingReplyStatus_t` in case pinging is not enabled.
But in order to get things rolling, I will remove the #ifdef for now.